pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Looks like you ran `pnpm install` with an older version of pnpm. Can you install the latest version and install dependencies again?